packit/cli/validate_config.py

```diff
@@ -51,6 +51,8 @@ def validate_config(config, path_or_url):
 
     PATH_OR_URL argument is a local path or a URL to a git repository with packit configuration file
     """
-    api = get_packit_api(config=config, local_project=path_or_url)
-    output = api.validate()
+    # we use PackageConfig.load_from_dict for the validation, hence we don't parse it here
+    api = PackitAPI(config, None)
```

Comment:
Please, use key-word args to see what is set to `None` from the first look. (And that the comment above is actually a reason for this. By now, it's does not look related to anything.)

Reply:
And we should really call/name service_config `service_config`, this is really misleading.